src/Header/Accept/FieldValuePart/AbstractFieldValuePart.php

```diff
@@ -69,31 +69,40 @@ protected function getInternalValues()
      */
     public function getTypeString()
     {
-        return $this->getInternalValues()->typeString;
+        return $this->typeString;
     }
 
     /**
      * @return float $priority
      */
     public function getPriority()
     {
-        return (float) $this->getInternalValues()->priority;
+        return (float) $this->priority;
     }
 
     /**
      * @return \stdClass $params
      */
     public function getParams()
     {
-        return (object) $this->getInternalValues()->params;
+        return (object) $this->params;
     }
 
     /**
      * @return string $raw
      */
     public function getRaw()
     {
-        return $this->getInternalValues()->raw;
+        return $this->raw;
+    }
+
+    /**
+     * @param string $key
+     * @return mixed
+     */
+    public function getInternalType($key)
+    {
+        return $this->getInternalValues()->$key;
```

Comment:
This is a new method; why is it added?

Reply:
Removed. I was trying to avoid using of magic methods.